central/complianceoperator/v2/scanconfigurations/service/convert.go

```diff
@@ -166,19 +180,31 @@ func convertStorageScanConfigToV2ScanStatus(ctx context.Context, scanConfig *sto
 		profiles = append(profiles, profile.GetProfileName())
 	}
 
+	var errors []string
+	bindings, err := bindingsDS.GetScanSettingBindings(ctx, search.NewQueryBuilder().
+		AddExactMatches(search.ComplianceOperatorScanConfigName, scanConfig.GetScanConfigName()).ProtoQuery())
+	if err != nil {
+		return nil, err
+	}
+
+	bindingError := getLatestBindingError(bindings[0].Status)
+	if bindingError != "" {
+		errors = append(errors, bindingError)
+	}
+
```

Comment:
Each cluster has it's own binding of the name of the configuration.  So the query should add cluster and I think this section should move into the for loop around line 200.  Make sense?